tests/test_str.py

```diff
@@ -41,7 +41,22 @@ def test_Format():
 
     assert obj[result] == "3,4"
 
+<<<<<<< HEAD
```

Comment:
Ahm... Merge is NOT resolved.

Reply:
Resolved now. Can you please review and tell me if it is resolved or still having a merge conflict?

Reply:
You can see for yourself at the bottom of this page if there is a merge conflict. Currently "This branch has no conflicts with the base branch"